test/dynamo/test_subclasses.py

```diff
@@ -314,7 +314,8 @@ def forward(self, l_x_):
         check_count_and_graph(2, 2, 2, expected_graph)
 
         try:
-            x = torch._to_functional_tensor(t_clone2, mirror_autograd_meta=True)
+            x = torch._to_functional_tensor(t_clone2)
+            torch._mirror_autograd_meta(t_clone2, x)
```

Comment:
Given the name, it's pretty ambiguous which direction the mirroring happens to.

Reply:
open to bikeshedding, but for now I'm going with `mirror_autograd_meta_to(src, dst)`